mr/src/main/java/org/elasticsearch/hadoop/rest/RestClient.java

```diff
@@ -410,7 +408,7 @@ public void refresh(Resource resource) {
         sb.setLength(sb.length() - 1);
         sb.append("\n]}");
 
-        if (!isES20) {
+        if (esVersion().equals(EsVersion.V_2_X)) {
```

Comment:
Actually, This should just be if the version is equal to `V_1_X`. This line is balancing the curly braces.